test/CadlRanchProjects/parameters/body-optionality/src/Generated/BodyOptionalityClient.cs

```diff
@@ -165,9 +165,9 @@ public virtual async Task<Response> RequiredImplicitAsync(string name, Cancellat
         {
             Argument.AssertNotNull(name, nameof(name));
 
-            BodyModel bodyModel = new BodyModel(name, null);
```

Comment:
Is this change due to the TCGC update in MGC?

Reply:
This is caused by cadl ranch update.